chain/beacon/drand/drand.go

```diff
@@ -190,8 +190,8 @@ func (db *DrandBeacon) VerifyEntry(curr types.BeaconEntry, prev types.BeaconEntr
 func (db *DrandBeacon) MaxBeaconRoundForEpoch(filEpoch abi.ChainEpoch, prevEntry types.BeaconEntry) uint64 {
 	// TODO: sometimes the genesis time for filecoin is zero and this goes negative
 	latestTs := ((uint64(filEpoch) * db.filRoundTime) + db.filGenTime) - db.filRoundTime
-	dround := (latestTs - db.drandGenTime) / uint64(db.interval.Seconds())
-	return dround
+	_, nextTs := dchain.NextRound(latestTs)
```

Comment:
Can you make this code explicit? I am afraid of for example "bugfix" to drand causing consensus  break.

Reply:
I could but the idea is that lotus should use the round - timestamp mapping that drands uses no ?

Reply:
Done